addons/hr_org_chart/static/src/fields/hr_org_chart.js

```diff
@@ -134,11 +135,11 @@ export class HrOrgChart extends Field {
             this.view_employee_id = null;
         } else if (employeeId !== this.view_employee_id || force) {
             this.view_employee_id = employeeId;
-            var orgData = await this.rpc(
+            var orgData = await this.rpc( // You should use orm.call. The user_ is hab by de
```

Comment:
Also `const` it needs to be an rpc since it's using a route and not a model method

Reply:
Ok my bad. you could use context: this.user.context for the user context :)